vllm/v1/worker/tpu_model_runner.py

```diff
@@ -797,6 +809,14 @@ def load_model(self) -> None:
                 "get_tensor_model_parallel_rank",
                 return_value=xm_tp_rank):
             model = get_model(vllm_config=self.vllm_config)
+        if self.lora_config is not None:
+            model = self.load_lora_model(model, self.model_config,
+                                         self.scheduler_config,
+                                         self.lora_config, self.device)
+            punica_wrapper = self.lora_manager._adapter_manager.punica_wrapper
+            if not self.enforce_eager:
```

Comment:
I think we can always `mark_compiled` even when `enforce_eager`. For TPU, the difference between `enforce_eager` True and False is whether we do pre-compilation or not.